sig-apps/agenda.md

```diff
@@ -22,7 +22,7 @@ _Note, the [minutes and agenda have moved to Google Docs](https://docs.google.co
 
 ## July 27, 2016 [[notes](minutes/2016-07-27.md)]
 * Intro
-* [Gerred Dillon](https://twitter.com/justicefries) will do a demo of [Deis](https://deis.com/)
+* [Gerred Dillon](https://twitter.com/justicefries) will do a demo of [Deis](https://azure.microsoft.com/en-us/services/kubernetes-service/)
```

Comment:
Would be better to use this link here: https://github.com/deis.

Reply:
the Deis link is moot because of the Microsoft acquisition. Deis is no more... 😢